src/md/bylaws.md

```diff
@@ -0,0 +1,381 @@
+project, the corresponding approval required for that action and those who have
+binding votes over the action.
+
+<table border="0" class="table table-striped">
+<thead>
+  <tr class="a">
+    <th>Action</th>
+
+    <th>Description</th>
+
+    <th>Approval</th>
+
+    <th>Binding Votes</th>
+  </tr>
+</thead>
+
+<tbody>
+  <tr class="b">
+    <td>Code Change</td>
+
+    <td>A change made to a codebase of the project and committed by a committer.
+    This includes source code, documentation, website content, etc.</td>
+
+    <td>Lazy approval and then Lazy consensus.</td>
+
+    <td>Active committers</td>
+  </tr>
+
+  <tr class="a">
+    <td>Major Feature/Change via a Branch Merge</td>
+
+    <td>A major change made to the codebase of the project done via a branch
+    merge.</td>
+
+    <td>Lazy consensus</td>
+
+    <td>Active committers</td>
+  </tr>
+
+  <tr class="b">
+    <td>Release Plan</td>
+
+    <td>Defines the timetable and actions for a release. The plan also nominates
+    a Release Manager.</td>
+
+    <td>Lazy majority</td>
+
+    <td>Active committers</td>
+  </tr>
+
+  <tr class="a">
+    <td>Product Release</td>
+
+    <td>When a release of one of the project&rsquo;s products is ready, a vote is
+    required to accept the release as an official release of the project.</td>
+
+    <td>Lazy Majority</td>
+
+    <td>Active PMC members</td>
+  </tr>
+
+  <tr class="b">
+    <td>Adoption of New Codebase</td>
+
+    <td>When the codebase for an existing, released product is to be replaced
+    with an alternative codebase. If such a vote fails to gain approval, the
+    existing code base will continue. This also covers the creation of new
+    sub-projects within the project.</td>
+
+    <td>2/3 majority</td>
+
+    <td>Active committers</td>
+  </tr>
+
+  <tr class="a">
+    <td>New Committer</td>
+
+    <td>When a new committer is proposed for the project</td>
+
+    <td>Lazy consensus</td>
+
+    <td>Active PMC members</td>
+  </tr>
+
+  <tr class="b">
+    <td>New PMC Member</td>
+
+    <td>When a committer is proposed for the PMC</td>
+
+    <td>Lazy consensus</td>
+
+    <td>Active PMC members</td>
+  </tr>
+
+  <tr class="a">
+    <td>Committer Removal</td>
+
+    <td>When removal of commit privileges is sought. Note: Such actions will also
+    be referred to the ASF board by the PMC chair</td>
+
+    <td>Consensus</td>
+
+    <td>Active PMC members (excluding the committer in question if a member of
+    the PMC).</td>
+  </tr>
+
+  <tr class="b">
+    <td>PMC Member Removal</td>
+
+    <td>When removal of a PMC member is sought. Note: Such actions will also be
+    referred to the ASF board by the PMC chair</td>
+
+    <td>Consensus</td>
+
+    <td>Active PMC members (excluding the member in question).</td>
+  </tr>
+
+  <tr class="a">
+    <td>Change to Project By-Laws</td>
+
+    <td>When a change is needed to the Project&rsquo;s By-Laws.</td>
+
+    <td>Lazy Consensus</td>
```

Comment:
Lazy consensus is fairly liberal for bylaws changes.  You should discuss in the community whether you want 2/3 majority here.